Assets/Python/RiseAndFall.py

```diff
@@ -478,25 +468,25 @@ def prepareChina(self):
 			pXian.setRouteType(-1)
 			pXian.setFeatureType(iForest, 0)
 			
-		if utils.getScenario() == i1700AD:
+		elif utils.getScenario() == i1700AD:
 			pBeijing = gc.getMap().plot(tBeijing[0], tBeijing[1])
 			pBeijing.getPlotCity().kill()
 			pBeijing.setImprovementType(-1)
 			pBeijing.setRouteType(-1)
 
 		tCultureRegionTL = (98, 37)
 		tCultureRegionBR = (109, 49)
-		for x in range(tCultureRegionTL[0], tCultureRegionBR[0]+1):       
-			for y in range(tCultureRegionTL[1], tCultureRegionBR[1]+1):     
+		for x in range(tCultureRegionTL[0], tCultureRegionBR[0]+1):

```

Comment:
I did do this, but somehow, I got an Unidentifiably C++ exception @ line 493,pCurrent.setOwner(-1).